maxim/logger/google_adk/client.py

```
@@ -372,557 +843,354 @@ async def on_tool_error_callback(
+                # Create and add Maxim plugin with callbacks
+                maxim_plugin = MaximInstrumentationPlugin(
+                    maxim_logger, 
+                    debug,
+                    before_generation_callback=_global_callbacks.get("before_generation"),
+                    after_generation_callback=_global_callbacks.get("after_generation"),
+                    before_trace_callback=_global_callbacks.get("before_trace"),
+                    after_trace_callback=_global_callbacks.get("after_trace"),
+                    before_span_callback=_global_callbacks.get("before_span"),
+                    after_span_callback=_global_callbacks.get("after_span"),
+                )
+                plugins = list(plugins) + [maxim_plugin]
```

Ruff (0.14.0)

Consider `[*list(plugins), maxim_plugin]` instead of concatenation

Replace with `[*list(plugins), maxim_plugin]`

(RUF005)